library/pkparse.c

```diff
@@ -1453,11 +1453,17 @@ int mbedtls_pk_parse_public_key( mbedtls_pk_context *ctx,
     if( ret == 0 )
     {
         p = pem.buf;
-        if( ( pk_info = mbedtls_pk_info_from_type( MBEDTLS_PK_RSA ) ) == NULL )
+        if( ( pk_info = mbedtls_pk_info_from_type( MBEDTLS_PK_RSA )) == NULL )
```

Comment:
minor: I don't know which is correct, but `))` differs from `) )` 6 lines down - as these are so close together they should be consistent

Reply:
Sure, I'll fix this with any other request changes to not re-trigger the CI now.